### PR TITLE
Ads shortlink + text update

### PR DESCRIPTION
**Pull request by keunes**

### CODE REVIEW

File: _i18n/en/documentation/general/ads.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-The official versions of AntennaPod are completely ad-free and open source! If you are seeing ads, this can have two reasons:
+The official versions of AntennaPod are completely ad-free and open source! If you are seeing or hearing ads, this can have two reasons:
 
-- **The podcast publishers added the ads to their shownotes or audio file.** AntennaPod plays podcasts just like they were published by the creators. If they include ads, AntennaPod plays them. We have nothing to do with those ads, so the revenue fully goes to the publishers of your podcast.
+- **The podcast creator or the company hosting the podcast included ads in the shownotes, chapter art or audio files.** AntennaPod gets episodes as they are published. If they include ads, AntennaPod plays or displays them. Some hosting companies have 'automatic ad insertion', which make ads sound injected in an unnatural way. The revenues fully go to the creators and/or the hosting company of your podcast.
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/ads.md:3]: Maybe making it more explicit by adding: "AntennaPod has nothing to do with the ads or their content."

**Comment by ByteHamster** [comment]
Google Play comments already link to this, which generates a 404 page @keunes

**Comment by ByteHamster** [comment]
Why did you close this? I think we should have the shortlink to not guide users to nowhere

**Comment by keunes** [comment]
Because of merge conflicts I didn't manage to resolve. I'm working on a new PR.

**Pull request closed without merging**